src/tools/prte/prte.c

```diff
@@ -463,8 +463,9 @@ int main(int argc, char *argv[])
     if (NULL != opt) {
         prte_pmix_server_globals.report_uri = strdup(opt->values[0]);
     }
-    /* don't aggregate help messages as that will apply job-to-job */
-    prte_setenv("PRTE_MCA_prte_base_help_aggregate", "0", true, &environ);
+
+    /* aggregate messages from prte. */
+    prte_setenv("PRTE_MCA_prte_base_help_aggregate", "1", true, &environ);
```

Comment:
Err...we just removed that MCA param, so why set it here? It is "on" by default for each job anyway.

Reply:
Right..this snuck through in the master PR. See https://github.com/openpmix/prrte/pull/1355